go/interpreter/lfvm/instructions.go

```diff
@@ -1119,44 +1127,64 @@ func opCall(c *context) {
 		cost += tosca.Gas(params.CallStipend)
 	}
 
-	// If we are in static mode, recursive calls are to be treated like
-	// static calls. This is a consequence of the unification of the
```

Comment:
Iff you move the static check somewhere else, please take this comment there as well.